README.md

```diff
@@ -22,10 +22,12 @@ This addon can help manage both Ember 1.x and 2.x deprecations.
 
 The initial steps needed to get started:
 
-1. Install the  ember-cli-deprecation-workflow addon.
-2. Run your test suite.*
-3. Run `deprecationWorkflow.flushDeprecations()` from your browsers console.
-4. Copy the string output into `config/deprecation-workflow.js` in your project.
+1. Install the ember-cli-deprecation-workflow addon (`ember install ember-cli-deprecation-workflow`).
+2. Run your test suite* with `ember test`.
```

Comment:
This should be ran via `ember test --server`, and should not require starting another server (if you start another server and do `.flushDeprecations()` you will not get the benefit of having your test suite building up the full list of deprecations).